classification2.Rmd

```diff
@@ -157,6 +157,7 @@ set.seed(1)
 random_numbers <- sample(0:9, 10, replace=TRUE)
 random_numbers
 
+set.seed(1)
```

Comment:
I don't think this is correct... why add this?

Reply:
So this is how the output looks without the change:
<img width="495" alt="Screen Shot 2022-04-19 at 11 42 08 AM" src="https://user-images.githubusercontent.com/3838091/164073624-f6a0afcc-649d-4806-b177-9d0b26a9799b.png">

So we get a different set of random numbers -- I imagine the intention is that we want to show the same?

Reply:
maybe the text wasn't clear -- the intention was to show that if you call sample twice (after set seed), you get the same two sequences of numbers in the same order. 

I would prefer not to call set seed twice, because then students might be tricked into thinking you have to set the seed before every function call that you want to be reproducible. 

happy to entertain more clear wording choice here!

Reply:
Oh, I see! Yeah okay, that wasn't clear to me, I thought you were trying to show that set.seed(1) would give the same set of numbers twice in a row. The first one is showing a set.seed. Then the second random_numbers is showing a different set. Then I thought the last two were a side by side comparison of the set.seed(1) to show the same sequence of numbers.  Shall we change the wording then? 
perhaps if we assign the variables as "random_numbers1", "random_numbers2", "random_numbers3" and "random_numbers4" 
then we can reference the set of numbers and say: 
"Notice that after setting the seed, we get the same two sequences of numbers in the same order. `random_numbers1` and `random_numbers3` produce the same sequence of numbers after setting the seed. The same can be said about `random_numbers2` and random_numbers4`."

Reply:
yup that works. but maybe make the link even clearer with names like `random_numbers1` and `random_numers1_again`
and `random_numbers2`, `random_numbers2_again`

Reply:
Perfect! Thanks!